config/metrics.js

```diff
@@ -113,6 +113,10 @@ module.exports = {
 		extract: (item) => !!getBPRules(item)?.allowsDeletions,
 		permissions: ['ADMIN'],
 	},
+	AllowsCocs: {
+		extract: (item) => !!getBPRules(item)?.AllowsCocs,
```

Comment:
is `AllowsCocs` a checkbox?

A code of conduct is considered "present" when the repo, or the owning user/org's `.github` repo, has a `CODE_OF_CONDUCT.md` file.

Reply:
not sure I understand correctly. So I need to check somehow if coc.md file does exist in repo?

Reply:
I'm not sure if github provides anything nicer in the API - the manual way is to check for the presence of the files, but since https://github.com/ljharb/repo-report/community provides a consolidated indicator of all of these checks, maybe the API reports it?

Reply:
we can use the Codes of Conduct API to retrieve information about a repository's code of conduct. To get a repository's code of conduct https://docs.github.com/en/rest/reference/codes-of-conduct#get-a-code-of-conduct. Here it is . I am just thinking how we can you it easily.